aule/visitors/alfaASTVisitor.py

```diff
@@ -61,7 +61,7 @@ def visitTargetStatement(self, ctx):
     def visitTargetLogicalExpression(self, ctx):
         left = self.visit(ctx.targetExpression(0))
         right = self.visit(ctx.targetExpression(1))
-        operator = ctx.logicalOperator().getText().upper()
+        operator = ctx.logicalOperator().getText().lower()
         return LogicalExpression(operator, left, right)
 
     def visitTargetUnaryExpression(self, ctx):
```

Comment:
`69 return UnaryExpression("NOT", left)`

Now `NOT` is the only operator in uppercase. Need to change to 
`69 return UnaryExpression("not", left)`

Reply:
:-(